common/src/main/java/com/datastrato/gravitino/dto/requests/TableUpdateRequest.java

```diff
@@ -141,8 +141,7 @@ public SetTablePropertyRequest() {
     public void validate() throws IllegalArgumentException {
       Preconditions.checkArgument(
           StringUtils.isNotBlank(property), "\"property\" field is required and cannot be empty");
-      Preconditions.checkArgument(
-          StringUtils.isNotBlank(value), "\"value\" field is required and cannot be empty");
+      Preconditions.checkArgument(value != null, "\"value\" field is required and cannot be null");
```

Comment:
I think you need to change the logic of `MetalakeUpdate`, `CatalogUpdate`, and so on.

Reply:
![image](https://github.com/datastrato/gravitino/assets/15794564/42121802-dce3-4077-b918-3a1fc16fcf31)

Reply:
Why do we need to allow `MetalakeUpdate` to set empty property value?

The table property needs an empty value because the Hive table requires `\t` to be set as the field delimiter. However, I did not see that Metalake needs such a scene.

Reply:
Since it's specified for the Hive table, is it proper to put it in the `TableUpdateRequest`?

If we apply this logic to the interface API, I believe that Matalake, catalog, and other entities will also be applied.

Reply:
> Since it's specified for the Hive table, is it proper to put it in the TableUpdateRequest?

Do you have a better solution?

> If we apply this logic to the interface API, I believe that Matalake, catalog, and other entities will also be applied.

There is no application scenario to do that.

@jerryshao What's your opinion on this?

Reply:
> Do you have a better solution?

Not yet, I only suggest not just changing `TableUpdateRequest`.

Reply:
I'm OK with the current change, we can gradually loose the constraints when there's a case.